rpi5.yaml

```diff
@@ -42,6 +48,58 @@ components:
       additional_deps:
         - "%{YOCTOS_WORK_DIR}/%{DOMD_BUILD_DIR}/tmp/deploy/images/%{MACHINE}/Image-%{MACHINE}.bin"
 
+  domu_sync:
+    default: true
+    build-dir: "%{ZEPHYR_DOMU1_DIR}"
+    sources:
+      - type: west
+        url: "https://github.com/xen-troops/zephyr.git"
+        rev: "rpi5_dev"
+
+    builder:
+      type: zephyr
+      board: xenvm
+      target: zephyr/samples/synchronization
+      work_dir: "%{ZEPHYR_DOMU1_BUILD_DIR}"
+      target_images:
+        - "%{ZEPHYR_DOMU1_BUILD_DIR}/zephyr/zephyr.bin"
+      additional_deps:
+        - "%{YOCTOS_WORK_DIR}/%{DOMD_BUILD_DIR}/tmp/deploy/images/%{MACHINE}/Image-%{MACHINE}.bin"
+
+  domu_blinky:
+    default: true
+    build-dir: "%{ZEPHYR_DOMU2_DIR}"
+    sources:
+      - type: west
+        url: "https://github.com/xen-troops/zephyr.git"
+        rev: "rpi5_dev"
+
+    builder:
+      type: zephyr
+      board: xenvm
+      target: zephyr/samples/basic/blinky
+      work_dir: "%{ZEPHYR_DOMU2_BUILD_DIR}"
+      snippets:
+        - "rpi_5_xen_domd"
+      target_images:
+        - "%{ZEPHYR_DOMU2_BUILD_DIR}/zephyr/zephyr.bin"
+      additional_deps:
+        - "%{YOCTOS_WORK_DIR}/%{DOMD_BUILD_DIR}/tmp/deploy/images/%{MACHINE}/Image-%{MACHINE}.bin"
+
+  domu_bins:
```

Comment:
@dsemenets  already works on the similar implementation. Please sync with him. https://github.com/xen-troops/meta-xt-prod-devel-rpi5/pull/9